tools/build

```diff
@@ -147,7 +168,9 @@ while [ $# -gt 0 ]; do
                  -p $pg_port \
                  --cache old-versions \
                  . \
-                 "$pg_config"
+                 "$pg_config" \
+		 "$cargo_pgx" \
+		 "$cargo_pgx_old"
```

Comment:
Line these up with the above.

Reply:
Apologies for being dense, but what does that look like?

```
                $nop cargo run --manifest-path tools/update-tester/Cargo.toml -- \
                 -h localhost \
                 -p $pg_port \
                 --cache old-versions \
                 . \
                 "$pg_config" \
		 "$cargo_pgx" \
		 "$cargo_pgx_old"
```
seems lined up to me?